shopinvader_payment_paypal/controllers/main.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Akretion (http://www.akretion.com)
+# Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo.addons.shopinvader.controllers import main
+from odoo.http import route
+
+_logger = logging.getLogger(__name__)
+
+
+class InvaderController(main.InvaderController):
+    @route(["/shopinvader/payment_paypal/normal_return"], methods=["GET"])
+    def normal_return(self, **params):
+        return self._process_method(
+            "payment_paypal", "normal_return", None, params
```

Comment:
```suggestion
            "payment_paypal", "normal_return", None, params=params
```

@qgroulard, can you try this ?

Reply:
That's the correct fix:
```suggestion
            "payment_paypal", "normal_return", params=params
```
Anyway this is deprecated, it's better to use decorators introduced with the new version of base_rest directly on the function normal_return in the payment_paypal service.